### ISSUE TITLE
Rework remote page

### ISSUE THREAD

**Issue by Julien-Blanc-tgcm** [Julien-Blanc-tgcm/kontroller#10]
The remote page has several issues :
* do not work in landscape mode
* « swipe-like » gestures are not practical with current settings
* misses the play / pause buttons.

**Comment by dikonov** [comment]
![remote_in_landscape](https://user-images.githubusercontent.com/4271322/78268627-1f679f80-7511-11ea-8e3b-0246aad4a88a.jpg)

In 0.5.1 the remote page still looks bad in landscape. The top icons are off-screen. The graphics looks quite blurry on a tablet. Where are the icons? I can make sharper versions.

Even in portrait mode there are layout issues: 
- the top 4 icons are shifted left
- the bottom part is empty, but it can easily house play/pause, stop, forward, rewind and mute/unmute buttons.
* Sometimes my ALSA audiodevice is occupied by another program and Kodi starts mute. It is not possible to unmute it without walking up to the computer and pressing keys there.
![remote-top_icons_too_far_left,empty_space_down](https://user-images.githubusercontent.com/4271322/78269160-dcf29280-7511-11ea-8af1-e6415fba918e.jpg)
Ideally, I would suggest adding a pull up menu with hot lists of audiotracks, subtitles and another with 3D modes + invert stereo option for video. Music paying might have stereo downmix and DTS passtrough/LPCM options.

**Comment by Julien-Blanc-tgcm** [comment]
Yes, this page needs a complete rework. I'm planning to merge it with the currently playing page, i think most of the operations is shared here.

Switching the audio device is not something you would do usually, so i don't think it belongs to this page. There is a need for a "kodi settings" page, though. (may be a child page of the remote). Library update is needed as well, for example.

**Comment by Michal-Szczepaniak** [comment]
@Julien-Blanc-tgcm i'll be redesigning and reworking entire gui (i'll make pr) so you might want to hold up on that issue

**Comment by Julien-Blanc-tgcm** [comment]
@Michal-Szczepaniak  That's good to hear. I'm currently working on server discovery (again after a holiday break), so i'll keep this on hold for a moment. Waiting for your PRs ! Thanks
